keps/sig-contributor-experience/0007-20180403-community-forum.md

```diff
@@ -0,0 +1,182 @@
+---
+kep-number: 0007
+title: A community forum for Kubernetes
+authors:
+  - "@castrojo"
+owning-sig: sig-contributor-experience
+participating-sigs:
+reviewers:
+  - "@jberkus"
+  - "@joebeda"
+  - "@cblecker"
+approvers:
+  - "@parispittman"
+  - "@grodrigues3"
+  
+editor: TBD
+creation-date: 2018-04-03
+last-updated: 2018-04-03
+status: provisional
+---
+
+# A community forum for Kubernetes
+
+(This is a draft and not PRed or proposed yet)
+
+## Table of Contents
+
+* [Table of Contents](#table-of-contents)
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non-Goals](#non-goals)
+* [Proposal](#proposal)
+    * [User Stories [optional]](#user-stories)
+      * [Story 1](#story-1)
+      * [Story 2](#story-2)
+    * [Implementation Details/Notes/Constraints](#implementation-details)
+    * [Risks and Mitigations](#risks-and-mitigations)
+* [Graduation Criteria](#graduation-criteria)
+* [Implementation History](#implementation-history)
+* [Drawbacks](#drawbacks)
+
+
+## Summary
+
+Kubernetes is large enough that we should take a more active role in growing our community. We need a place to call our own that can encompass users, contributors, meetups, and other groups in the community. Is need for something inbetween email and real time chat that can fulfill this? The primary purpose of this KEP is to determine whether we can provide a better community forum experience and perhaps improve our mailing list workflow. 
+
+The site would be forum.k8s.io, and would be linked to from the homepage and major properties. [See KEP005](https://github.com/kubernetes/community/blob/master/keps/sig-contributor-experience/0005-contributor-site.md) for related information on a contributor website. 
+
+## Motivation
+
+- We're losing too much information in the Slack ether, and most of it does not show up in search engines. 
+- Mailing lists remain mostly the domain of existing developers and require subscribing, whereas an open forum allows people to drive by and participate with minimal effort.
+  - There's an entire universe of users and developers that we could be reaching that didn't grow up on mailing lists and emacs. :D
+  - Specifically, hosting our lists on google groups has some issues:
```

Comment:
tacking on a +1 here: google groups is also limited to strictly google accounts. I cannot use my @microsoft.com domain to subscribe to the mailing list, for example. However, I know that's being changed with a slow roll-out to [groups.io](https://lists.cncf.io/g/main/subgroups).